.github/workflows/codecov.yml

```diff
@@ -33,4 +35,4 @@ jobs:
           directory: packages/aws-cdk/coverage
           fail_ci_if_error: true
           flags: suite.unit
-          token: ${{ secrets.CODECOV_TOKEN }}
+          use_oidc: true
```

Comment:
Switch from token based authentication to OIDC. Verified to work [here](https://github.com/cdklabs/cdk-from-cfn/blob/main/.github/workflows/status-checks.yml#L79-L85). This is really nice as it allows us to entirely remove the need for a long lived token.